nifi-registry-core/nifi-registry-web-ui/src/main/webapp/components/administration/workflow/sidenav/manage-bucket/nf-registry-manage-bucket.html

```diff
@@ -41,6 +41,20 @@
                 Save
             </button>
         </div>
+        <div class="pad-top-md pad-bottom-sm pad-left-md pad-right-md">
+            <div class="pad-bottom-sm">
+                <span class="md-card-title">Bundle Settings</span>
+            </div>
+            <div>
+                <mat-checkbox #bundleRedeployCheckbox
+                              [disabled]="!nfRegistryService.currentUser.resourcePermissions.buckets.canWrite"
+                              [(checked)]="nfRegistryService.bucket.allowBundleRedeploy"
+                              [(ngModel)]="allowBundleRedeploy"
```

Comment:
I found a slight bug in this PR. If you edit a bucket and check to "Allow bundle overwrite" for said bucket... and then you click the 'X' in the top right of the sidenav... and then you click the browser's back navigation button... you will see that the checkbox selection is incorrect (it should be checked at this point). To fix this please remove the ```[(checked)]="nfRegistryService.bucket.allowBundleRedeploy"``` and update the ```[(ngModel)]="nfRegistryService.bucket.allowBundleRedeploy"```